### ISSUE TITLE
Add error detail before stackTrace in PageReportMode

### ISSUE THREAD

**Issue by narumishi** [jhomlala/catcher#163]
In page report mode, only stackTrace is shown, but the error not. It's helpful to show them together.
https://github.com/jhomlala/catcher/blob/0d03422643a45b440eebfc0408204d5f86469ad5/lib/mode/page_report_mode.dart#L151-L171

Select one non-nullable value of `error` and `errorDetails`. Something like this
```dart
final List<String> items = [
        if (widget.report.error != null) widget.report.error.toString(),
        if (widget.report.error == null && widget.report.errorDetails != null)
          widget.report.errorDetails.toString(),
        ...widget.report.stackTrace.toString().split("\n"),
      ];
```

**Comment by jhomlala** [comment]
Fixed in 0.6.4.

**Comment by narumishi** [comment]
Please add error detail in error widget too. 
`_buildStackTraceWidget` inside `catcher_error_widget.dart`
